pkg/flowaggregator/flowaggregator.go

```diff
@@ -482,18 +485,11 @@ func (fa *flowAggregator) fillK8sMetadata(key ipfixintermediate.FlowKey, record
 	}
 }
 
-func (fa *flowAggregator) fetchPodLabels(podAddress string) string {
-	pods, err := fa.podInformer.Informer().GetIndexer().ByIndex(podInfoIndex, podAddress)
+func (fa *flowAggregator) fetchPodLabels(podName string, podNamespace string) string {
+	pod, err := fa.podLister.Pods(podNamespace).Get(podName)
 	if err != nil {
 		klog.Warning(err)
```

Comment:
for the functions you update, could you make sure all the `klog` calls are changed to use structured logging
In this case, use `klog.InfoS` or `klog.ErrorS` as you deem appropriate

Reply:
Sure, updated.